.github/workflows/shippable_builds.yml

```diff
@@ -74,17 +81,20 @@ jobs:
             const matrixApk = matrixFull.filter(item => item.packageFormat == "apk");
             core.setOutput("matrixIncludeApk", matrixApk);
             core.setOutput("matrixInclude", matrixFull);
+            core.setOutput("releaseVersion", releaseVersion);
```

Comment:
If you are not processing the release version in any way, you can simply reference `github.event.inputs.releaseVersion` anywhere you need to in this file, you don't need to make it an output of the environment step or dump_config job.